deepface/modules/preprocessing.py

```diff
@@ -88,13 +88,12 @@ def load_base64(uri: str) -> np.ndarray:
     encoded_data = encoded_data_parts[1]
     decoded_bytes = base64.b64decode(encoded_data)
 
-    img = Image.open(io.BytesIO(decoded_bytes))
-    file_type = img.format.lower()
-
     # similar to find functionality, we are just considering these extensions
     # content type is safer option than file extension
-    if file_type not in ["jpeg", "png"]:
-        raise ValueError(f"input image can be jpg or png, but it is {file_type}")
+    with Image.open(io.BytesIO(decoded_bytes)) as img:
```

Comment:
This is actually not needed as in this case we're not opening a file handle: all data is already in memory (decoded_bytes). Hence the auto closure in a `with` block has no effect.

Reply:
Out of curiosity: why are you proxying `decoded_bytes` through `io.BytesIO`
The signature of Image.open allows to pass directly the `decoded_bytes` bytes object

Reply:
You mean `Image.open(decoded_bytes)` performs same? TBH, ChatGPT showed me that usage.

Reply:
For with block, agree it has no effect but still i believe it is good practice to confirm opened things closed.

Reply:
> You mean `Image.open(decoded_bytes)` performs same? TBH, ChatGPT showed me that usage.

Never ever trust ChatGPT blindly ... I beg you.
It can give you some hints but **never** let AI write code for you unless you have **rock solid certainty** it is correct, reliable and best practice.
Please.